spec/tout_spec.rb

```diff
@@ -27,6 +35,14 @@ def private_foo
       end
     end
 
+    context "for a private class method" do
+      publicize(NormalClass, :private_class_foo)
+
+      it "makes the class method public" do
+        expect(NormalClass.private_class_foo).to eq("private class")
+      end
+    end
+
     context "raises an error when given a public method" do
```

Comment:
@nficca Can we add test cases for checking privacy of the class method in the after block, and the other failure cases that are outline in the existing test suite? ie: when the private class method does not exist, when the class method is already public...

Thanks!

Reply:
@aarongough I think testing to make sure that the given private class method exists is redundant, since the test for that would be identical to the test that checks the existence of a public method.

Otherwise, done :)